### ISSUE TITLE
Trakt may have synced episode plays from far in the future

### ISSUE THREAD

**Issue by ArthurZey** [trakt/Plex-Trakt-Scrobbler#558]
The issue is described in great detail in the Plex Forums:
["Recently Played Episodes" show episodes played over a year in the future](https://forums.plex.tv/t/recently-played-episodes-show-episodes-played-over-a-year-in-the-future/580050)

I suspect that it's the Trakt.tv plugin that caused this, but I can't be sure. If someone could help me to diagnose the matter to see whether it's really the plugin, I would be very grateful!

**Comment by agneevX** [comment]
Yeah this could be a possible issue with how the database is updated. 

Generally, it makes sense to update info in Trakt and then let it sync to your Plex server.
